### PR TITLE
Feature/update moving endpoint

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

**Comment by brettminnie** [comment]
I know it close couples... do you think I could get ruby to send anything that resembled an array that we could consume :(

File: src/TransformCore/PHE/HayApi/PersistenceBundle/Controller/MovingController.php [hunk 1]
```diff
@@ -37,24 +37,78 @@ protected function getService()
...
+     * )
+     * @RequestParam(
+     *  name="strengthening_activity_durations_sat",
+     *  default=0,
+     *  description="Duration in minutes of saturday strengthening activity"
+     * )
+     * @RequestParam(
+     *  name="strengthening_activity_durations_sun",
+     *  default=0,
+     *  description="Duration in minutes of sunday strengthening activity"
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Controller/MovingController.php]: Wow thats growing :(

Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Controller/MovingController.php]: I appreciate time is a constraint at the moment

File: src/TransformCore/PHE/HayApi/PersistenceBundle/Entity/Questions/Moving.php [hunk 2]
```diff
@@ -111,14 +116,16 @@ public function getAerobicActivityDays()
     }
 
     /**
-     * @param array $aerobicActivityDays
+     * @param ParamFetcher $paramFetcher
      * @return Moving
      */
-    public function setAerobicActivityDays(array $aerobicActivityDays)
+    public function setAerobicActivityDays(ParamFetcher $paramFetcher)
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Entity/Questions/Moving.php]: Coupling Entity to FOSRest

File: src/TransformCore/PHE/HayApi/PersistenceBundle/Controller/MovingController.php (hunk 1 above)
Comment (brettminnie) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Controller/MovingController.php]: Yeah, can't get the blasted array to get passed in a format that FOSRest accepts, have tried for 2 days :/

Basically have exploded the array, I have flagged this as Tech Debt

File: src/TransformCore/PHE/HayApi/PersistenceBundle/Entity/Questions/Moving.php (hunk 2 above)
Comment (brettminnie) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Entity/Questions/Moving.php]: Noted, see problem above, is related. See #43

**Pull request merged**